website/src/app/kb/user-guides/linux-client/readme.mdx

```diff
@@ -35,17 +35,18 @@ Once you have a token, you can start the Linux Client using the following
 command:
 
 ```
-sudo ./linux-client-x64 <TOKEN>
+sudo FIREZONE_TOKEN=<TOKEN> ./linux-client-x64
```

Comment:
CLI args are very easy to snoop, even for root processes. In this case we are still putting it in the command line to make it a one-liner, but I'd rather get rid of this example entirely.

Reply:
Yeah it's a tradeoff. At the least it could be prefixed with a space char to prevent storing the command in history

Reply:
We do the same thing when deploying Gateways... not a great workaround other than having the user manually create the file with token, so no one-liner. Might be worth updating that.

What if the instructions were this instead:

```
# Save token to file. Note the leading space to prevent this command from being saved to history
 echo "<TOKEN>" > .firezone-token

# Export needed vars
export FIREZONE_NAME="Development Webserver"
export FIREZONE_ID="some unique identifier"
export DNS_CONTROL="systemd-resolved" # or "etc-resolv-conf"
export LOG_DIR="./"

# Start Firezone client
sudo -E ./linux-client-x64
```

Reply:
Hm. I think I'll leave it for now. When https://github.com/firezone/firezone/issues/4664 is closed (should be easy) the KB can just recommend using that to point at a local token file instead of the `/etc` one.

Since it runs with root I did program it to be really picky about ownership and permissions, but I can put that into the example... hopefully nobody minds, I think Wireguard already has a precedent that it'll refuse to touch privkey files if you forget to do `chmod 400` on them.